ext/standard/rand.c

```diff
@@ -258,32 +258,6 @@ PHP_FUNCTION(mt_srand)
 /* }}} */
 
 
-/*
- * A bit of tricky math here.  We want to avoid using a modulus because
- * that simply tosses the high-order bits and might skew the distribution
- * of random values over the range.  Instead we map the range directly.
- *
- * We need to map the range from 0...M evenly to the range a...b
- * Let n = the random number and n' = the mapped random number
- *
- * Then we have: n' = a + n(b-a)/M
- *
- * We have a problem here in that only n==M will get mapped to b which
- # means the chances of getting b is much much less than getting any of
- # the other values in the range.  We can fix this by increasing our range
- # artificially and using:
- #
- #               n' = a + n(b-a+1)/M
- *
- # Now we only have a problem if n==M which would cause us to produce a
- # number of b+1 which would be bad.  So we bump M up by one to make sure
- # this will never happen, and the final algorithm looks like this:
- #
- #               n' = a + n(b-a+1)/(M+1)
- *
- * -RL
- */
-
```

Comment:
@datibbaw Because the implementation of the algorithm the comment describes is in the [header file, lines 69-70](https://github.com/bishopb/php-src/blob/464bdd45c44c9a6260ff21b90e25b999c43e7193/ext/standard/php_rand.h#L69-L70).